packages/valibot/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "@rvf/valibot",
+	"version": "6.0.0",
+	"main": "./dist/index.cjs.js",
+	"module": "./dist/index.esm.js",
+	"types": "./dist/index.d.ts",
+	"license": "MIT",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/airjp73/remix-validated-form"
+	},
+	"scripts": {
+		"dev": "tsup --watch",
+		"build": "tsup",
+		"prepublishOnly": "npm run build",
+		"typecheck": "tsc --noEmit"
+	},
+	"peerDependencies": {
+		"@rvf/core": ">= 0.0.0 < 7.0.0",
+		"@rvf/set-get": ">= 0.0.0 < 1.0.0",
+		"valibot": ">= 0.31.0"
+	},
+	"devDependencies": {
+		"@rvf/yup": "*",
```

Comment:
I don't think we need to depend on the yup package here. `@rvf/zod` does just because that's where the all-adapters tests are.